### ISSUE TITLE
Request: Provide XPI files for new releases

### ISSUE THREAD

**Issue by RibShark** [insin/control-panel-for-twitter#288]
Given how many new versions of this are being released currently, and how slow the Firefox add-ons repo can be at approving new extensions, it would be great if you could release a [self-distributed XPI](https://extensionworkshop.com/documentation/publish/submitting-an-add-on/#self-distribution) file for new releases so people can update quickly without having to resort to using the less user-friendly userscript version.

**Comment by Zachruff** [comment]
hi rib

**Comment by Cpt-KeksMonster** [comment]
Yes, please supply the XPI as FF extension repo is still only offering 3.4.1.

**Comment by insin** [comment]
The problem with this is:

1. [version numbers have to be unique across listed and unlisted versions](https://blog.mozilla.org/addons/2017/01/26/mixing-listed-and-unlisted-on-amo/)
2. Firefox doesn't support using any kind of `-pre` or `-beta` prefix in version numbers for the unlisted version which could then be superceded by the listed version

We've [asked @mozamo how version numbers work](https://twitter.com/ControlPanelFT/status/1710897709924880608) - if they work out the way we want we might be able to support creating unlisted versions.

If not, we don't want to complicate our release process further or need to have have weird multi-version number bumps for all 220,000+ users just to support whatever % of the ~35,000 Firefox users would make use of this build.
